### PR TITLE
fix(cli/lint): don't use gray in diagnostics output for visibility

### PR DESCRIPTION
**Pull request by magurotuna**
This commit changes output color of the linter to not use gray for visibility on dark backgrounds.

Fixes https://github.com/denoland/deno_lint/issues/788

### CODE REVIEW

File: cli/tools/lint.rs [hunk 1]
```diff
@@ -308,9 +307,9 @@ pub fn format_diagnostic(
       message_line,
```
Comment (bartlomieju) [inline cli/tools/lint.rs:307]: While you looking into it, could you rewrite this conditional? Lots of duplication and not very readable.

Comment (magurotuna) [reply cli/tools/lint.rs]: Sure, how about this? 99a83ff

Comment (bartlomieju) [reply cli/tools/lint.rs]: That works, thanks!

**Review by bartlomieju — APPROVED** [review]
LGTM

**Pull request merged**